baseplate/experiments/providers/feature_flag.py

```diff
@@ -7,7 +7,9 @@
 
 
 class FeatureFlag(R2Experiment):
-    """An experiment with a single variant "active".
+    """An experiment with a single variant "active". This type of experiment
```

Comment:
might be useful to link to the design doc rationale for why.  (For example, I don't know)

Reply:
This repo is OS, so can't really link internal docs.

Reply:
Reason for deprecation? Just because it's redundant. I'd like to stop supporting the legacy bucketing altogether for simplicity if possible.

Reply:
I'm not opposed to deprecating this to try to simplify the config format/keep things more consistent.  The "legacy" bucketing doesn't really affect anything here since there's only one variant, it can be any size.

Reply:
My thinking is that it's basically a feature rollout, with 100%, but with specific targeting. That's going to be replaced with the feature_rollout option once targeting is in place (work in progress)